src/Template/Layout/js/app/components/edit-relation-params.js

```diff
@@ -1,40 +1,170 @@
+            <div>
+                <span class="tag" :class="objectColorClass"><: object.type :></span>
+                <span class="tag"><: object.attributes.status :></span>
+            </div>
+
+            <h2><: t('Related to') :>: <: relatedName :></h2>
+            <div>
+                <span class="tag" :class="relatedColorClass"><: related.type :></span>
+                <span class="tag"><: relatedStatus :></span>
+            </div>
+        </div>
+
+        <form ref="paramsForm" class="params-list" @change="checkParams()" @keyup="checkParams()">
+            <div>
+                <label><: t('Priority') :></label>
+                <input type="number" step="1" v-model.number="priority" />
+            </div>
+
+            <div v-for="(param, key) in schema">
+                <label :title="key"><: param.description || key :>
+
+                    <!-- Boolean -> switch button true/false --->
+                    <div v-if="param.type == 'boolean'">
+                        <label class="switch">
+                            <input type="checkbox" v-model="editingParams[key] ">
+                            <span class="slider round"></span>
+                        </label>
+                    </div>
+
+                    <div v-if="param.type == 'string'">
+                        <!-- String, format: date-time -> datepicker --->
+                        <input v-if="param.format == 'date-time'" v-datepicker time="true" v-model="editingParams[key]"></input>
+
+                        <!-- String Enum -> select/option --->
+                        <div v-else-if="param.enum !== undefined">
+                            <select v-model="editingParams[key]">
+                                <option v-for="item in param.enum" :value="item"><: item :></option>
+                            </select>
+                        </div>
+                        <!-- String -> input text --->
+                        <input v-else type="text" v-model="editingParams[key]">
+                    </div>
+
+                    <!-- Number -> input number --->
+                    <div v-if="param.type == 'number'">
+                        <label>
+                            <input type="number" :name="key" step="any" v-model.number="editingParams[key]">
+                        </label>
+                    </div>
+
+                    <!-- Integer -> input integer --->
+                    <div v-if="param.type == 'integer'">
+                        <label>
+                            <input type="number" :name="key" v-model.number="editingParams[key]">
+                        </label>
+                    </div>
+                </label>
+            </div>
+        </form>
+
+        <footer>
+            <p>
+                <button v-show="isModified"
+                    class="has-background-info has-text-white"
+                    @click.prevent="saveParams()"><: t('Save ') :></button>
```

Comment:
Moving view from twig to js, translation references changed.
Update can be done in some steps:

 - launch `bin/cake gettext update`
 - edit properly `*.po` files
 - commit & push

Reply:
fixed, although i noticed that some core properties don't get translated (title, body, lang) we might wanna check into that eventually

Reply:
good point: we should put those words (that are not mentioned directly inside templates) in a specific file, `config/i18n_strings.php`